Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -273,8 +273,13 @@ - (void)testCacheDirectoryPath
 
 - (NSString *)getDefaultCacheDirectoryPath
 {
+    // #if TARGET_OS_OSX
```

Comment:
`h`: Not sure why this commented out. Please remove it if not used anymore, or add a comment why it is commented out.